docs/products/kafka/kafka-connect/howto/jdbc-source-connector-pg.rst

```diff
@@ -0,0 +1,113 @@
+JDBC source connector with PostgreSQL
+=====================================
+
+The JDBC source connector pushes data from a relational database, such as PostgreSQL, to Apache Kafka where can be transformed and read by multiple consumers. 
+
+.. Tip::
+
+    Sourcing data from a database into Apache Kafka decouples the database from the set of consumers, enabling multiple applications to access the data without adding a per-consumer query overhead to the source database.
+
+.. _connect_jdbc_pg_source_prereq:
+
+Prerequisites
+-------------
+
+To setup a JDBC source connector pointing to PostgreSQL, you need an Aiven for Apache Kafka service :doc:`with Kafka Connect enabled <enable-connect>` or a :ref:`dedicated Aiven for Apache Kafka Connect cluster <apache_kafka_connect_dedicated_cluster>`. 
+Furthermore you need to collect the following information about the source PostgreSQL database upfront:
+
+* ``PG_HOST``: The database hostname
+* ``PG_PORT``: The database port
+* ``PG_USER``: The database user to connect
+* ``PG_PASSWORD``: The database password for the ``PG_USER``
+* ``PG_DATABASE_NAME``: the database name
+* ``SSL_MODE``: the `SSL mode <https://www.postgresql.org/docs/current/libpq-ssl.html>`_
+* ``PG_TABLES``: the list of database tables to be included in Apache Kafka; the list must be in the form of ``schema_name1.table_name1,schema_name2.table_name2``
+
+.. Note::
+
+    If you're using Aiven for PostgreSQL the above details are available in the `Aiven console <https://console.aiven.io/>`_ service Overview tab or via the dedicated ``avn service get`` command with the :ref:`Aiven CLI <avn_service_get>`.
```

Comment:
We need to mention somewhere that topics will not be created automatically unless it is enabled. Since we don't sow any errors, it will be confusing for the users that they have a connector, but data is simply not coming.

Reply:
added a tip